pyriemann/utils/mean.py

```diff
@@ -14,12 +14,12 @@
 def mean_ale(covmats, tol=10e-7, maxiter=50, sample_weight=None):
```

Comment:
to go all the way covmats param could be renamed but that's a deprecation cycle whichi is never fun...

Reply:
Yes, my sweetest dream is to rename all `covmats` into `X` !
We could plan to group all deprecations in a release. What do you think? @agramfort @sylvchev

Reply:
@agramfort , what is the best way to do this deprecation for module `utils.mean`?
Renaming argument `covmats` into `X`?
Renaming argument and transforming it into a positional-only argument?

Reply:
example with new signature:

def mean_ale(X, tol=10e-7, maxiter=50, sample_weight=None, covmats=None):

then if covmats is not None you raise a deprecation warning that covmats has been renamed to X and that this break in the next release.

clear?

Reply:
Yes, it's clear and elegant. Thx!